notebooks/DAAC_UWG_cloud_tutorial/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -0,0 +1,430 @@
+    "\n",
+    "The ATL06 version 005 collection is identified by the `concept_id` C2153572614-NSIDC_CPRD.  The temporal range is identified with standard date strings, and latitude-longitude corners of a bounding box is specified.  Polygons and points, as well as shapefiles can also be specified."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3e8b65d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query = earthaccess.granule_query().concept_id(\n",
+    "    'C2153572614-NSIDC_CPRD'\n",
+    ").temporal(\n",
+    "    \"2020-03-01\", \"2020-03-30\"\n",
+    ").bounding_box(\n",
+    "    -134.7,58.9,-133.9,59.2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc6924e3",
+   "metadata": {},
+   "source": [
+    "As before, we can use `Query.hits()` to find the number of granules that match our search. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "15163612",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query.hits()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f65f1e6",
+   "metadata": {},
+   "source": [
+    "We'll get metadata for these 4 granules and display it.  The rendered metadata shows a download link, granule size and two images of the data.  _This doesn't render so well on this Jupyter Hub._\n",
+    "\n",
+    "The download link is `https` and can be used download the granule to your local machine.  This is similar to downloading _DAAC-hosted_ data but in this case the data are coming from the Earthdata Cloud.  For NASA data in the Earthdata Cloud, there is no charge to the user for egress from AWS Cloud servers.  This is not the case for other data in the cloud."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4680671a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "granules = Query.get(4)\n",
+    "[display(g) for g in granules]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3664c14b",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Use Direct-Access to open, load and display data stored on S3\n",
+    "\n",
+    "Direct-access to data from a S3 bucket, is a two step process.  First, the files are opened using the `open` method.  The `auth` object created at the start of the notebook is used to provide Earthdata Login authentication and AWS credentials.\n",
+    "\n",
+    "The next step is to load the data.  In this case, data are loaded into an `xarray.Dataset`.  Data could be read into `numpy` arrays or a `pandas.Dataframe`.  However, each granule would have to be read using an package that reads HDF5 granules such as `h5py`.  `xarray` does this all _under-the-hood_ in a single line but for a single group in the HDF5 granule, in this case land ice heights for the gt1l beam*.\n",
+    "\n",
+    "*ICESat-2 measures photon returns from 3 beam pairs numbered 1, 2 and 3 that each consist of a left and a right beam*"
```

Comment:
Do we want to keep the '*' before "ICESat-2" (following "beam" above)? I think it is formatting to italicized because there is another '*' at the end of the sentence. 
```suggestion
    "*ICESat-2 measures photon returns from 3 beam pairs numbered 1, 2 and 3 that each consist of a left and a right beam"
```